ts/features/itwallet/common/utils/itwCryptoContextUtils.ts

```diff
@@ -0,0 +1,26 @@
+import { deleteKey, generate } from "@pagopa/io-react-native-crypto";
+import { constNull } from "fp-ts/lib/function";
+
+// eID key tags
+export const WIA_EID_KEYTAG = "WIA_EID_KEYTAG";
+export const DPOP_EID_KEYTAG = "DPOP_EID_KEYTAG";
+
+// Credentials key tags
+export const WIA_CREDENTIAL_KEYTAG = "WIA_CREDENTIAL_KEYTAG";
+export const DPOP_CREDENTIAL_KEYTAG = "DPOP_CREDENTIAL_KEYTAG";
+
+export const regenerateCryptoKey = (keyTag: string) =>
+  deleteKey(keyTag)
+    .catch(constNull)
+    .finally(() => generate(keyTag));
+
+export const disposeEidCryptoKeys = () =>
+  Promise.all([deleteKey(WIA_EID_KEYTAG), deleteKey(DPOP_EID_KEYTAG)]).catch(
+    constNull
+  );
+
+export const disposeCredentialCryptoKeys = () =>
+  Promise.all([
+    deleteKey(WIA_CREDENTIAL_KEYTAG),
+    deleteKey(DPOP_CREDENTIAL_KEYTAG)
+  ]).catch(constNull);
```

Comment:
Questions: 
- Do we really need to dispose these keys given that we are just regenerating them when needed? 
- Nitpicking but Isn't something like `disposeCredentialEphemeralryptoKeys` more appropriate? We could add a comment as well to explain what they do and why. 

Let me know what you think

Reply:
You are right, those keys are fixed and we delete them before getting new WIA/DPoP keys. I removed that code as well in 7d1038c25a288edfd17f382fea8fb73654e7173b.